_posts/2021-10-01-debezium-server-iceberg.adoc

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title:  Using Debezium to Create ACID Data Lake
+date:   2021-10-01
+tags: [ debezium, iceberg, datalake, lakehouse ]
+author: isimsek
+---
+
+Do you need to build Lakehouse with near realtime data pipeline, do you want it to support ACID transactions and updates on data lake?
```

Comment:
I think we need set the context a bit here, not all the Debezium users are familiar with the concepts of data lakes and such. Heck, I'm not even sure myself what "lake house" means in this context :)